hazelcast/src/main/java/com/hazelcast/config/AbstractXmlConfigHelper.java

```diff
@@ -22,6 +22,7 @@
 import com.hazelcast.logging.ILogger;
```

Comment:
Can't we move this class to impl?

Reply:
Same as above, public API ends up inheriting from this class, thus making public methods on this class public API.